addons/mail/security/ir.model.access.csv

```diff
@@ -46,4 +46,5 @@ access_mail_resend_message,access.mail.resend.message,model_mail_resend_message,
 access_mail_resend_partner,access.mail.resend.partner,model_mail_resend_partner,base.group_user,1,1,1,0
 access_mail_template_preview,access.mail.template.preview,model_mail_template_preview,base.group_user,1,1,1,0
 access_mail_blacklist_remove_system,acesss.mail.blacklist.remove.system,model_mail_blacklist_remove,base.group_system,1,1,1,1
+access_mail_user_settings,mail.user.settings,model_mail_user_settings,base.group_user,1,1,1,1

```

Comment:
You also need to add rules for "no group" (probably just 0s).

Reply:
added